code/obj/artifacts/artifactanalysis.dm

```diff
@@ -183,3 +183,14 @@
 			var/obj/O = src.attached
 			O.remove_suffixes("\[[src.artifactType]\]")
 			O.UpdateName()
+
+	mouse_drop(atom/over_object, src_location, over_location, over_control, params)
+		if(!istype(usr, /mob/living) || !isturf(src.loc) || \
+				BOUNDS_DIST(get_turf(over_object), get_turf(src)) > 0 || \
+				BOUNDS_DIST(usr, get_turf(over_object)) > 0 ||  \
+				BOUNDS_DIST(usr, src) > 0 || \
+				over_object == usr || !istype(over_object, /atom/movable))
```

Comment:
```suggestion
				over_object == usr || !istype(over_object, /atom/movable))
```
why can't borgs stick artifact sheets to themselves

Reply:
No reason in particular, just something that was part of the barcode code I used for this, gonna change remove it.